src/examples/solana_list_rewards.ts

```diff
@@ -0,0 +1,18 @@
+import { StakingReward } from "../coinbase/staking_reward";
+import { Coinbase } from "../coinbase/coinbase";
+import { Address } from "../coinbase/address";
+
+async function printStakingInfo() {
+  Coinbase.configureFromJson({ filePath: "~/Downloads/cdp_api_key.json" });
```

Comment:
Can we either:
1. Accept this as a cmd line param, or
2. Leave it as `<path_to_api_key>`?

I think including a hardcoded path here can be misleading for new developers.

Reply:
Sounds good 👍